lib/MemoryHandler.js

```diff
@@ -133,3 +157,7 @@ MemoryStore.prototype._sortList = function(request, list) {
     }
   });
 };
+
+MemoryStore.prototype._clone = function(obj) {
+  return JSON.parse(JSON.stringify(obj));
```

Comment:
Since we don't really need the `MemoryStore` object instance here, I am wondering if this shouldn't simply me a module level helper function, i.e., `MemoryStore._clone`.